scripts/templates/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -287,6 +289,20 @@ spec:
               port: 8071
     ```
 
+    ## Adding Certificates for Your Own Trusted Certificate Authorities
+
+    Specify a set of certificates to be imported into the Java trust store
+    of the running IQ Server.
+
+    ```yaml
+    iq:
+      certificates:
+        my-company-cert: |
+          -----BEGIN CERTIFICATE-----
+          ...
+          -----END CERTIFICATE-----
+    ```
+
```

Comment:
specifics of how to use the new `certificates` param.

Reply:
This is very useful. 

Just out of curiosity, is there a way to pass the certificate as a file in here, not as string literal? Some customers may ask for that.

Reply:
INT-6831 is _very_ similar to this story, but will allow the ability to mount existing storage or config maps that have the CER files.